src/library_gl.js

```diff
@@ -715,9 +715,22 @@ var LibraryGL = {
     switch(name_) {
       case 0x1F00 /* GL_VENDOR */:
       case 0x1F01 /* GL_RENDERER */:
-      case 0x1F02 /* GL_VERSION */:
         ret = allocate(intArrayFromString(GLctx.getParameter(name_)), 'i8', ALLOC_NORMAL);
         break;
+      case 0x1F02 /* GL_VERSION */:
+        var glVersion = GLctx.getParameter(GLctx.VERSION);
+        // return GLES version string corresponding to the version of the WebGL context
+#if USE_WEBGL2
+        if (GLctx.canvas.GLctxObject.version >= 2) {
```

Comment:
While this is technically correct (if we went to initialize a WebGL 2 context, this field will have been set), I'd prefer not to depend on this here, but instead directly examine what the `.getParameter` returned us, and use that to transform the result to the GLES variant. That will be more future proof since `GLctx.canvas.GLctxObject.version` is an Emscripten-specific "cached" field, and `GLctx.getParameter` is more authoritative "live" information.